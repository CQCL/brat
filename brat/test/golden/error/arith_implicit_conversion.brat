f(Int, Float) -> Int
f(x, y) = x + (x * y)
