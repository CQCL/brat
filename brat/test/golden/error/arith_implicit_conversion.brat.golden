Error in test/golden/error/arith_implicit_conversion.brat@2:20-21
f(x, y) = x + (x * y)
                   ^

  Type mismatch when checking y
Expected: (right :: Int)
But got:  (y :: Float)


